cpp/oneapi/dal/backend/common.hpp

```diff
@@ -357,6 +357,20 @@ inline std::int64_t propose_wg_size(const sycl::queue& q) {
     return std::min<std::int64_t>(512, device_max_wg_size(q));
 }
 
+inline std::int64_t get_scaled_wg_size_per_row(const sycl::queue& queue,
```

Comment:
I suggest to add doxygen for that function, it's hard to understand what it does

Reply:
Done